### PR TITLE
Implemented `draw` method for `HexGridView`

### PR DESCRIPTION
**Pull request by notthatmj**

### CODE REVIEW

File: Hex Paper/HexGridView.swift [hunk 1]
```diff
@@ -8,19 +8,143 @@
...
+        }
+        
+        let leftVertex = lattice.pointAt(row: centerPointRow, column: centerPointColumn - 1)
+        let topLeftVertex = lattice.pointAt(row: centerPointRow - 1, column: centerPointColumn - 1 + columnOffset)
+        let topRightVertex = lattice.pointAt(row: centerPointRow - 1, column: centerPointColumn + columnOffset)
+        let bottomLeftVertex = lattice.pointAt(row: centerPointRow + 1, column: centerPointColumn - 1 + columnOffset)
+        let bottomRightVertex = lattice.pointAt(row: centerPointRow + 1, column: centerPointColumn + columnOffset)
+        let rightVertex = lattice.pointAt(row: centerPointRow, column: centerPointColumn + 1)
+        
+        return Set( [Edge(leftVertex, topLeftVertex),
```
Comment (jacobvanorder) [inline Hex Paper/HexGridView.swift]: Extra space.

File: Hex Paper/HexGridView.swift [hunk 2]
```diff
@@ -8,19 +8,143 @@
...
+                     Edge(bottomRightVertex, rightVertex),
+                     ])
+    }
+}
+
+@IBDesignable
+class HexGridView: UIView {
+    
+    @IBInspectable var hexRadius: CGFloat = 10.0
+    @IBInspectable var gridColor: UIColor = UIColor.gray
```
Comment (jacobvanorder) [inline Hex Paper/HexGridView.swift]: You don't need `UIColor` on the right operand.

**Comment by jacobvanorder** [comment]
Minor stuff. Looks good!

**Review by jacobvanorder — APPROVED** [review]

**Comment by notthatmj** [comment]
Good points. Thanks!

**Pull request merged**